tests/unit/recommenders/models/test_embdotbias_utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+from recommenders.models.embdotbias.utils import cartesian_product
+
+
+def test_cartesian_product_two_arrays():
+    a = np.array([1, 2])
+    b = np.array([3, 4])
+    result = cartesian_product(a, b)
+    expected = np.array([[1, 3], [1, 4], [2, 3], [2, 4]])
+    np.testing.assert_array_equal(result, expected)
```

Comment:
@jmarrietar this is good. the only thing missing would be the tests for the model. That would include the data loader, the trainer, the fit, score methods, etc. With a very small dataset. There are two ways of doing this, either create one test file called `test_embdotbias.py` or creating two files: `test_embdotbias_utils.py` and `test_embdotbias_model.py`. See these tests for some reference: https://github.com/recommenders-team/recommenders/tree/main/tests/unit/recommenders/models